samples/authorization/csm_authorization_proxy_server_v230.yaml

```diff
@@ -92,7 +92,6 @@ metadata:
   namespace: authorization
 data:
   csm-config-params.yaml: |-
-    CONCURRENT_POWERFLEX_REQUESTS: 10
-    CONCURRENT_POWERSCALE_REQUESTS: 10
+    CONCURRENT_STORAGE_REQUESTS: 10
```

Comment:
Do we not have a default if the parameter is not provided? If we do then perhaps a test file could exercise that default?